concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md

```diff
@@ -97,6 +97,83 @@ The main idea is to filter the data source by the `parentID` for different drill
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ...
+            :data-source="dxDataSource">
```

Comment:
Why use the 'dx' prefix? I suggest removing it and use 'dataSource' here and below

Reply:
It's case for using DevExpress.data.DataSource. I'll fix other cases

Reply:
Fixed